src/DynamoCoreWpf/UI/Themes/Modern/LiveChartsStyle.xaml

```diff
@@ -63,17 +63,17 @@
     </system:Double>
 
     <Style TargetType="lvc:CartesianChart" x:Key="CartesianLiveChart">
-        <Setter Property="AnimationsSpeed" Value="0:0:0.420"></Setter>
-        <Setter Property="SeriesColors" Value="{StaticResource ColorsCollection}"/>
+        <Setter Property="AnimationsSpeed" Value="00:00:00.420"></Setter>
+        <!--<Setter Property="SeriesColors" Value="{StaticResource ColorsCollection}"/>-->
```

Comment:
Are these commented code legacy that we could remove?

Reply:
will remove